build.gradle

```diff
@@ -1756,6 +1756,7 @@ project(':tools') {
 
   dependencies {
     implementation project(':clients')
+    implementation project(':core')
```

Comment:
It would be bit harder to move them to tools. For example the command uses `kafka.utils.json` which is used by `kafka.server`, `kafka.security.authorizer`. And `kafka.admin.AdminOperationException` is used by `kafka.admin`, `kafka.controller`, `kafka.zk`, etc. So not sure it's easy to move them out.

Reply:
These classes may be candidates to land in the module `server-common`?

Reply:
Let's avoid making tools depend on core if possible. I opened https://issues.apache.org/jira/browse/KAFKA-14737 to move the kafka.utils.json classes to server-common.

Reply:
I removed the dependencies on the core now. Out of curiosity do we plan to move `kafka.utils.TestUtils` out of core and maybe into common at some point?